sdk/Sdk.Generators/FunctionMetadataProviderGenerator.Parser.cs

```diff
@@ -504,7 +504,7 @@ private bool TryGetAttributeProperties(AttributeData attributeData, Location att
                 {
                     if (namedArgument.Value.Value != null)
                     {
-                        if (string.Equals(namedArgument.Key, Constants.IsBatchedKey) && !attrProperties.Keys.Any(k => string.Equals(k, "Cardinality", StringComparison.OrdinalIgnoreCase)))
+                        if (String.Equals(namedArgument.Key, Constants.IsBatchedKey) && !attrProperties.Keys.Contains("Cardinality"))
```

Comment:
nit: lower case `string` was a nice to have change:
```suggestion
                        if (string.Equals(namedArgument.Key, Constants.IsBatchedKey) && !attrProperties.Keys.Contains("Cardinality"))
```